README.md

```diff
@@ -1,6 +1,46 @@
 # Job Manager
+See also the [development guide] below(#development)
```

Comment:
Maybe we could use either `[development guide below](#development)` or `[development guide](#development) below` to make the anchor link works.

Reply:
No don't worry, it wasn't the period, it was because I didn't keep the () near the [] part in a very silly move!! Good catch here!